backend/src/test/java/com/redhat/cloud/notifications/routers/AuthenticationTest.java

```diff
@@ -67,8 +66,7 @@ void testEndpointRoles() {
                 .header(identityHeader)
                 .when().get("/endpoints")
                 .then()
-                .statusCode(403)
-                .contentType(JSON);
+                .statusCode(403);
```

Comment:
would this test fail if the order would stay the same?

Reply:
Quarkus 2 no longer returns a content type when the returned status is 403, that's why the content type check has to be removed.

Reply:
ah.. now i get it. you removed the contenttype.. i thought you moved the lines around.. ok, makes sense to me.